### DISCUSSION TITLE
Can I use GUI at blinkshell?

### DISCUSSION THREAD

**Discussion by kamgaa** [blinksh/blink#1482] [Q&A]
I'm using ROS on ssh server, and want to implement robot simulator or gedit which use gui . But everytime I try to do it, I encountered "Gtk-WARNING **: 07:41:54.747: cannot open display: :0" on my screen. TO handle the problum, I search for a month. However, every solutions are to use the X server not one the Blinkshell. So I wanna know if I can use a program that uses GUI. I know you're busy, but I want you to let me know. Just tell me it's available

**Comment by carloscabanero** [comment]
We do not have a X server, but you still have plenty of options. There are now HTTP implementations that will allow you to use it over the web. And you also have VNC clients that would make that easy.
